vms/platformvm/txs/builder/camino_builder.go

```diff
@@ -226,6 +276,94 @@ func (b *caminoBuilder) NewAddAddressStateTx(
 	return tx, tx.SyntacticVerify(b.ctx)
 }
 
+func (b *caminoBuilder) NewDepositTx(
```

Comment:
Shouldn't we have the duration as argument here, too? I think it ll be always 0 when creating the tx from the builder.

Reply:
you'r right, forgot to add that after we decided that duration won't be fixed